bash.sh

```
@@ -0,0 +1,7 @@
+
+trap "rm -rf '$(cd "$(mktemp -d)" && pwd)'" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)